assets/sass/_variables.scss

```diff
@@ -29,37 +27,62 @@ $color-attention: #cf0c4e;
+
+$color-arbitrary-background-b1: rgb(144, 202, 249); // Blue
+$color-arbitrary-background-b2: rgb(197, 225, 165); // Green
+$color-arbitrary-background-b3: rgb(255, 183, 77);  // Orange
+$color-arbitrary-background-b4: rgb(255, 241, 118); // Yellow
+$color-arbitrary-background-b5: rgb(158, 134, 201); // Purple
+$color-arbitrary-background-b6: rgb(229, 115, 115); // Red
+$color-arbitrary-background-b7: rgb(244, 143, 177); // Pink
+$color-arbitrary-background-b8: rgb(230, 230, 230); // Grey
+
 // Typography
 $font-primary: Georgia, serif;
 $font-secondary: "Avenir Next Webfont", Arial, Helvetica, sans-serif;
 $font-monospace: "Courier 10 Pitch", Courier, monospace;
-$font-size-base-in-px: 16;
-$font-size-h1-in-px: 36;
-$font-size-h2-in-px: 26;
-$font-size-h3-in-px: 22;
-$font-size-h4-in-px: 20;
-$font-size-h5-in-px: 18;
-$font-size-h6-in-px: 16;
-$font-size-caption-in-px: 13;
-$box-font-scaling-factor: 0.85;
 
-/*
- * Baseline grid
- */
-$blg-space-extra-small-in-px: 12;
-$blg-space-small-in-px: $blg-space-extra-small-in-px * 2;
-$blg-space-smallish-in-px: $blg-space-small-in-px * 1.5;
-$blg-space-medium-in-px: $blg-space-small-in-px * 2;
-$blg-space-large-in-px: $blg-space-small-in-px * 3;
-$blg-space-extra-large-in-px: $blg-space-small-in-px * 5;
+// TODO: Review with design whether explicitly tying together the baseline grid and the base font size is desireable.
+// It is a design requirement that most element heights are set using baseline grid variables. This requires the baseline
+// grid measure to scale when the base font size changes to avoid vertically overlapping lines of text.
+// The line below explicitly derives the base font size from the smallest baseline grid measure. This may be
+// too explicit, it may be enough to set them manually, keeping in mind their relationship.
+$font-size-base-in-px: $blg-measure-extra-small-in-px * 4 / 3;
+
+$font-size-small-in-px: $font-size-base-in-px * (14 / 16);
+$font-size-extra-small-in-px: $font-size-base-in-px * (11 / 16);
+
+$font-size-h1-in-px: $font-size-base-in-px * 36 / 16;
+$font-size-h2-in-px: $font-size-base-in-px * 26 / 16;
+$font-size-h3-in-px: $font-size-base-in-px * 22 / 16;
+$font-size-h4-in-px: $font-size-base-in-px * 20 / 16;
+$font-size-h5-in-px: $font-size-base-in-px * 18 / 16;
+$font-size-h6-in-px: $font-size-base-in-px;
+
+$font-size-caption-in-px: $font-size-base-in-px * 13 / 16;
+$font-size-table-text-in-px: $font-size-base-in-px * 13 / 16;
+
+// TODO: Create typographical style mixins from these? Can they be consolidated?
+$font-size-pullquote-in-px: $font-size-base-in-px * 24 / 16;
+$font-size-msa-link: $font-size-base-in-px * 12 / 16;
+$font-size-profile-name: $font-size-base-in-px * 18 / 16;
+$font-size-main-listing-article-title: $font-size-base-in-px * 20 / 16;
+$font-size-side-listing-article-title: $font-size-base-in-px * 18 / 16;
+$font-size-carousel-item-title: $font-size-base-in-px * 24 / 16;
+$font-size-content-header-simple-title-in-px: 20;
```

Comment:
Need to define typographical styles and consolidate these many values where possible.